frontend/src/app/visualization/barchart/timeline.component.html

```diff
@@ -31,8 +31,9 @@
     <div class="message-body">
         <div class="block">
         The results for your search include too many documents to count all matches.
-        Only the first {{percentageDocumentsSearched}}% of documents for each bin are analyzed;
-        the rest are assumed to have only one match for your query.
+        Only a sample of the first {{percentageDocumentsSearched}}% of documents for each bin are analyzed;
+        the rest are assumed to contain linearly decreasing matches between the end of the sample
+        to one match.
```

Comment:
Perhaps just change this into "the match count for other documents is extrapolated" and link to the manual for a full description.